wireapp/wire-android
# Feature: Integrate About screen and Support screen into Scala code

## What's new in this PR?

### Issues

Integrate and release screens in smaller compact chunks as a release train rather than a mammoth feature in one go. 

### Causes

We were using fragments to inflate layouts and behaviour

### Solutions

Now we can communicate from Scala to Kotlin using activities and remove the old code. 

### Testing

About screen: 
1. Open up the app
2. Open profile 
3. Go to settings 
4. Click about 
5. All buttons should work as previously intended

Support Screen: 
1. Open up the app
2. Open profile 
3. Go to settings 
4. Click support 
5. All buttons should work as previously intended

## Notes

QA will need to respond by changing the locators within their tests, so this may break stuff in the nightly build. 
#### APK
[Download build #1254](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1254/artifact/build/artifact/wire-dev-PR2611-1254.apk)
[Download build #1294](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1294/artifact/build/artifact/wire-dev-PR2611-1294.apk)
[Download build #1295](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1295/artifact/build/artifact/wire-dev-PR2611-1295.apk)
[Download build #1315](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1315/artifact/build/artifact/wire-dev-PR2611-1315.apk)
[Download build #1322](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1322/artifact/build/artifact/wire-dev-PR2611-1322.apk)
